ui/src/tool-panel/location-track/splitting/location-track-split.tsx

```diff
@@ -85,6 +85,7 @@ export const LocationTrackSplit: React.FC<SplitProps> = ({
         split.descriptionBase !== '',
     );
 
+    // TODO: Adding any kind of dependency array causes infinite re-render loops, find out why
```

Comment:
Tässä tosiaan on jotain melko kosmista, mainitsinkin tästä toisella reviewillä. Antamalla tälle `useEffect`:ille mitä tahansa depsuja tämä menee ennen pitkää rerender-looppiin. Pitäisi selvittää miksi näin käy, mutta en ehtinyt tekemään sitä tässä ennen lomaa

Reply:
Tuolle voisi tehdä vaikka tiksun tai jotain ettei unohdu tänne (ellet ole jo katsomassa). Se ei näytä yhtään selkeämmältä pystymetsästä tulevalle koodarille joka tähän törmää.